File: Web/RMA-Web-Shell/projects/fincare/src/app/shared/models/collection-agent.ts
import { User } from 'projects/shared-models-lib/src/lib/security/user';

export class CollectionAgent {
  agent: User;
  accountIds: number[];
  clientTypeId: number;
  ageTypeId: number;
  debtorStatus: number;
  assignedStatus: number;
  balanceTypeId: number;
  endDate: string;
}

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/CollectionAgent.cs
using RMA.Service.Admin.SecurityManager.Contracts.Entities;

using System;
using System.Collections.Generic;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class CollectionAgent
    {
        public User Agent { get; set; }
        public List<int> AccountIds { get; set; }
        public int ClientTypeId { get; set; }
        public int AgeTypeId { get; set; }
        public int BalanceTypeId { get; set; }
        public DateTime EndDate { get; set; }
    }
}
